arrow/arrow.py

```diff
@@ -1787,6 +1788,42 @@ def _get_tzinfo(tz_expr: Optional[TZ_EXPR]) -> dt_tzinfo:
             except parser.ParserError:
                 raise ValueError(f"{tz_expr!r} not recognized as a timezone.")
 
+    @classmethod
+    def excel_date(
+        cls, delta: Union[int, float], default_windows_date: bool = True
```

Comment:
Should we change the kwarg to instead use the Microsoft terminology of 1900 and 1904 datesystem: https://docs.microsoft.com/en-us/office/troubleshoot/excel/1900-and-1904-date-system? Like make the kwarg `1904_date_system=True`? Or do you think using the OS names is clearer?

Reply:
I think it makes more sense to use the 1900/1904 convention. I looked in Excel and that's what they end up of using.